src/extension/mapml/src/main/java/org/geoserver/mapml/MapMLDocumentBuilder.java

```diff
@@ -355,6 +355,7 @@ private MapMLLayerMetadata layersToOneMapMLLayerMetadata(List<RawLayer> layers)
         MapMLLayerMetadata mapMLLayerMetadata = new MapMLLayerMetadata();
         mapMLLayerMetadata.setLayerMeta(new MetadataMap());
         mapMLLayerMetadata.setUseTiles(false);
+        useFeaturesAllLayers = allLayersUsingFeatures(layers);
```

Comment:
I believe this one was meant to be gone, or not?

Reply:
fixed